waveorder/models/isotropic_thin_3d.py

```diff
@@ -169,7 +168,7 @@ def visualize_transfer_function(
             contrast_limits=(-lim, lim),
             scale=(1, 1, 1),
         )
-    viewer.dims.order = (0, 1, 2)
+    viewer.dims.order = (2, 0, 1)
```

Comment:
Why is this one not using the util function?

Reply:
Great question. This transfer function is our only 3d-to-2d transfer function, so it stands alone for now. I've also added a comment addressing this.